src/gui/log/loglistview.cpp

```diff
@@ -52,14 +56,21 @@ namespace
 
     QString logText(const QModelIndex &index)
     {
-        return u"%1%2%3"_qs.arg(index.data(BaseLogModel::TimeRole).toString(), SEPARATOR
-                                , index.data(BaseLogModel::MessageRole).toString());
+        return index.data(BaseLogModel::TimeRole).toString()
+            + SEPARATOR
+            + index.data(BaseLogModel::MessageRole).toString();
     }
 
     class LogItemDelegate final : public QStyledItemDelegate
     {
     public:
-        using QStyledItemDelegate::QStyledItemDelegate;
+        explicit LogItemDelegate(QObject *parent = nullptr)
+            : QStyledItemDelegate(parent)
+#ifdef Q_OS_WIN
+            , m_useCustomUITheme(Preferences::instance()->useCustomUITheme())
```

Comment:
(Non blocking) It would be better if this info is provided by `UIThemeManager`.

Reply:
I thought about it, but it will become strange if it can be queried but cannot set it via `UIThemeManager`. So for now, I would keep it as is.

Reply:
> I thought about it, but it will become strange if it can be queried but cannot set it via `UIThemeManager`.

My reasoning is about way to query effective value. The corresponding setting can be `true` but custom theme isn't actually used for some reason (e.g. missing/corrupted files).